### DISCUSSION TITLE
Possible bug 5.2.0

### DISCUSSION THREAD

**Discussion by JanOlsmar** [oqtane/oqtane.framework#4474] [Q&A]
I try to make a Theme switcher (night/day) in a Theme. So I needed interactive mode.
I have new 5.2.0 source code and develope a Theme from there.
But if I switch to interactive in UI section in site setting I get an error ....
Error: Found malformed component comment at Blazor:{"type":"server","prerenderId":"ea739dcf6542433cbf36deeb680d9048","key":{"locationHash":"476C06752106D8A3A05D54B0A17EF32D21166A00467A6703EAB4C70B27093295:11","formattedComponentKey":""},"sequence":0,"descriptor": ........
In blazor.web.js:1 .

I need to change the prerendering to NO to get it to work. (prerendering is default for interactive mode in Blazor)

I have a running site upgraded to 5.2.0 there I don't get this error.

Is it just me or .......

Thanks in advance

**Comment by sbwalker** [comment]
@JanOlsmar Can you please clarify the steps to reproduce the issue? Are you saying that when you have a Site which is configured to use Static render mode and you change it to Interactive render mode in Site Settings (and save the change), an exception occurs (and is the exception in the browser, in the logs, etc... do you have a screen shot you can share)?

**Reply by JanOlsmar** [reply]
To reproduce -  Oqtante source code  - load the site - change UI to interactive - stop site - load the site again

**Reply by sbwalker** [reply]
What do you mean by "stop site"?

**Reply by JanOlsmar** [reply]
Close site either you stop debugging or close browser

**Reply by sbwalker** [reply]
Ok so you are saying this is a not a production problem... it is only occurring during development. I will investigate.

**Reply by JanOlsmar** [reply]
Have not tested in production - new install.

**Reply by JanOlsmar** [reply]
Tested now - works fine in new install.

**Reply by sbwalker** [reply]
This is most likely related to the Visual Studio debugger.

**Reply by JanOlsmar** [reply]
Anyhow it is annoying. And you have to set prerender to false when you develop as the Theme does not show up ells.
